analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/RequestsWithExcessiveLength/Corrupt/Web.config

```diff
@@ -0,0 +1,10 @@
+﻿<configuration>
+  <system.web>
+    <httpRuntime maxRequestLength="81920" executionTimeout="3600" />
+    <!-- Sensitive: maxRequestLength is exprimed in KB, so 81920KB = 80MB  -->
```

Comment:
could you please remove the `Sensitive` comments? Because I have to think if this means an issue is found in the corrupt file